src/package-linker.js

```diff
@@ -380,15 +379,15 @@ export default class PackageLinker {
     }
   }
 
-  determineTopLevelBinLinks(flatTree: HoistManifestTuples): HoistManifestTuples {
+  determineTopLevelBinLinks(flatTree: HoistManifestTuples): Array<[string, Manifest]> {
```

Comment:
No but this function was only used once below, where it just pulled out the Manifest (`pkg`), and I thought it'd be better to reduce the size of object being passed from the HoistManifest. If you think we need to keep the previous signature, happy to send a refactor PR.